modules/serverless-setting-traffic-distribution-by-revisions.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: PROCEDURE
+[id="serverless-setting-traffic-distribution-by-revisions_{context}"]
+= Setting traffic distribution by revisions
```

Comment:
I would say that this whole topic about setting traffic distribution is not necessary in this document. 
The doc is about integration with OpenShift Pipelines and setting traffic distribution has nothing to do with it.
We already talk about traffic split in other parts of the docs: https://68520--docspreview.netlify.app/openshift-serverless/latest/knative-serving/traffic-splitting/traffic-splitting-revisions
It would probably make sense to remove this part.